gulpfile.js

```diff
@@ -130,6 +133,21 @@ gulp.task( 'bundle:generate',
 
 // Documentation. -------------------------------------------------------------
 
-const docsBuilder = ckeditor5DevCompiler.docs( config );
+// Documentation. -------------------------------------------------------------
+
+const ckeditor5DevDocs = require( '@ckeditor/ckeditor5-dev-docs' );
+const docsBuilder = ckeditor5DevDocs.docs( config );
+
+gulp.task( 'docs', [ 'compile:js:esnext' ], ( done ) => {
```

Comment:
To avoid the same mistake as we did with the other tasks, we should work already on minimising the number of sub tasks which we need to create.

So, the ckeditor5-dev-docs should expose a single method which will make for the `docs` task. All the rest (the sub tasks) can still be exposed by that package (so if anyone will need them, they can use it) but they should not be necessary to register here. They need to be used internally by the function which makes for the `docs` task.